auv_control.orogen

```diff
@@ -6,6 +6,7 @@ import_types_from "6dControl.hpp"
 using_library "motor_controller"
 
 typekit.export_types '/motor_controller/PIDSettings'
+typekit.export_types '/base/LinearAngular6DCommand'
```

Comment:
I'm not entirely sure, but I think so, yes.

What I originally meant was that if you get a type from e.g. `import_types_from "base"` then orogen projects that do `import_types_from 'auv_control'` won't necessarily get the types from 'base' as well. I actually thought it was final at the time.

As far as I understood, you were asking because you wanted to have a smooth transition, that is make sure that orogen projects that were importing the type from auv_control could still do so. I honestly did not think of explicitely exporting the type the way you do it right now. Which, from a quick look at the code, should actually work.

The question now is: do we want to keep this level of backward compatibility? Is anyone "using" the auv_control typekit apart from us ?